x/group/msgs_test.go

```diff
@@ -575,9 +575,42 @@ func TestMsgCreateGroupPolicy(t *testing.T) {
 			"expected a positive decimal",
 		},
 		{
-			"valid test case",
+			"invalid voting period",
+			func() *group.MsgCreateGroupPolicy {
+				policy := group.NewThresholdDecisionPolicy("-1", time.Duration(0), 0)
+				req, err := group.NewMsgCreateGroupPolicy(admin, 1, []byte("metadata"), policy)
+				require.NoError(t, err)
+				return req
+			},
+			true,
+			"expected a positive decimal",
+		},
+		{
+			"invalid execution period",
+			func() *group.MsgCreateGroupPolicy {
+				policy := group.NewThresholdDecisionPolicy("-1", time.Minute, 0)
+				req, err := group.NewMsgCreateGroupPolicy(admin, 1, []byte("metadata"), policy)
+				require.NoError(t, err)
+				return req
+			},
+			true,
+			"expected a positive decimal",
+		},
+		{
+			"valid test case, only voting period",
+			func() *group.MsgCreateGroupPolicy {
+				policy := group.NewThresholdDecisionPolicy("1", time.Second, 0)
+				req, err := group.NewMsgCreateGroupPolicy(admin, 1, []byte("metadata"), policy)
+				require.NoError(t, err)
+				return req
+			},
+			false,
+			"",
+		},
+		{
+			"valid test case, voting and execuion",
 			func() *group.MsgCreateGroupPolicy {
-				policy := group.NewThresholdDecisionPolicy("1", time.Second)
+				policy := group.NewThresholdDecisionPolicy("1", time.Second, 0)
```

Comment:
should we use a non zero minExecutionPeriod value here?

Reply:
I think that's okay

Reply:
Added new test case with non-empty minExecutionPeriod